.github/workflows/test-snap-can-build.yml

```diff
@@ -0,0 +1,25 @@
+name: 🧪 Test snap can be built on x86_64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]    
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: snapcore/action-build@v1
+        id: build
+
+      - uses: diddlesnaps/snapcraft-review-action@v1
+        with:
+          snap: ${{ steps.build.outputs.snap }}
+          isClassic: 'false'
```

Comment:
Maybe we can move that as a separate job in https://github.com/lapce/lapce/tree/master/.github/workflows/ci/yml ?

Reply:
We don't have to include these with the PR. They were intended to help you with managing the snap. These all can be omitted - including the other items above.